doc/source/user_guide/label_guide.rst

```diff
@@ -4,42 +4,46 @@
 Label guide
 ===========
 
-Basic labeling
---------------
+Basic labelling
+---------------
 
-All ArviZ plotting functions and some stats functions take an optional ``labeller`` argument.
-By default, labels show the variable name and the coordinate value
-(for multidimensional variables only).
-The first example below uses this default labeling.
+All ArviZ plotting functions and some stats functions can take an optional ``labeller`` argument.
+By default, labels show the variable name.
+Multidimensional variables also show the coordinate value.
 
 .. ipython::
 
+Example: Default labelling
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
   In [1]: import arviz as az
      ...: schools = az.load_arviz_data("centered_eight")
      ...: az.summary(schools)
 
-Thanks to being powered by xarray, ArviZ supports label based indexing.
-We can therefore use the labels we have seen in the summary to plot only a subset of the variables,
-the one we are interested in.
-Provided we know that the coordinate values shown for theta correspond to the `school` dimension,
-we can plot only ``tau`` to better inspect it's 1.03 :func:`~arviz.rhat` and
-``theta`` for ``Choate`` and ``St. Paul's``, the ones with higher means:
+ArviZ supports label based indexing.
```

Comment:
I would like to still be explicit about xarray. Many users only use ArviZ functions, plots and diagnostics and are not aware that all of that is powered by xarray under the hood. This is particularly relevant for researchers who then don't cite xarray when publishing even though they did cite ArviZ. It is not "our" problem per se, but I think we should try to nudge users towards the right citations and it is also bad for us indirectly because xarray is generally undercited which dificults it getting grants and slows down its development.